chukwuemekachm/GraphQL-API
# #11 Setup Prettier on the repository

#### What does this PR do?
- This PR completes the setup of Prettier on the repository
#### Description of Task to be completed?
- Add a prettier configuration
- Install the neccessary modules
- Add a `pre-commit` hook
#### How should this be manually tested?
- Update files on the repository
- Open the terminal and run `git add .`
- Run `git commit` to see prettier in action
#### Any background context you want to provide?
- N/A
#### What are the relevant github issues?
- [#11](https://github.com/chukwuemekachm/GraphQL-API/issues/11)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
